colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<p>I would split out the part where you set the config from the part where you give it to <code>tune</code>.  That's a chance to talk about what goes in a config, link to their docs, link to our <code>wandb.init</code>  docs.</p><p> </p><p>Lines 4 &amp; 6: I think we should drop these newlines.</p><p>Line 11: Why is <code>monitor_gym</code>  here? There's no gym.</p><p>Line 11: <code>raytune</code>  rather than <code>rayTune</code> .</p><p>Lines 14-15: These are nice targets for a <code>grid_search</code>,  but the search needs some adjustment. First, you duplicate the <code>lr</code> argument <code>0.1</code>  twice. <code>0.1</code>  is the one true learning rate, but it shouldn't be repeated. Also, they're not in order and there's no obvious rule to their spacing. This applies to <code>momentum</code>  as well. Finally, the search is too long: 24 runs is too many just for a demo.</p><p> </p><p>I suggest you switch to:</p><p> </p><pre class="ql-syntax" spellcheck="false">&nbsp;&nbsp;&nbsp;&nbsp;"lr": tune.grid_search([0.0001, 0.001, 0.1]),
&nbsp;&nbsp;&nbsp;&nbsp;"momentum": tune.grid_search([0.9, 0.99])
</pre><p> </p><p>Don't forget those spaces after the commas!&nbsp;&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>